### DISCUSSION TITLE
Resources and nothing

### DISCUSSION THREAD

**Discussion by emanuelbertey** [FlamxGames/godot-ai-assistant-hub#3] [Q&A]
I set everything up but there is no way for godot to let me edit a resource, not in .three assistants it is not editable, use the ones in the example and the alert menu it is impossible to do something without guidance, some help or idea, thanks

**Comment by FlamxGames** [comment]
Hi, can you share a screenshot? I'm not sure what you mean. If you do right click in a folder > Create New > Resource, select AIAssistantResource, save the resource, then you double click it, you should be able to edit its properties in the Inspector panel.

**Comment by FlamxGames** [comment]
Cleaning up old discussion.
